### PR TITLE
consolidate Current Spell and Water Step feats and update Elementalist Journal

### PR DESCRIPTION
**Pull request by ammalagonc**
- Feat has the same wording on the body of the text.
- Now adding Water Step to the list.
- Updated Elementalist Journal Entry to match update in Rage of Elements

### CODE REVIEW

File: packs/feats/current-spell.json [hunk 1]
```diff
@@ -63,6 +70,7 @@
         "traits": {
             "rarity": "common",
             "value": [
+                "archetype",
```
Comment (TikaelSol) [inline packs/feats/current-spell.json:73]: Not sure about adding the trait actually. Need to figure out how we want to handle these I think.

Comment (ammalagonc) [reply packs/feats/current-spell.json:73]: Want me to remove `archetype` trait?

Comment (ammalagonc) [reply packs/feats/current-spell.json]: PR updated - you should now see the two feats removed and the updated Journal Entry.

**Comment by ammalagonc** [comment]
Close to re-do

**Pull request closed without merging**